grafana-plugin/src/pages/users/Users.tsx

```diff
@@ -250,6 +230,37 @@ class Users extends React.Component<UsersProps, UsersState> {
     );
   }
 
+  renderFilters() {
+    const { query, store, theme } = this.props;
+    const styles = getUsersStyles(theme);
+
+    return (
+      <div className={styles.filters}>
+        <RemoteFilters
+          query={query}
+          page={PAGE.Users}
+          grafanaTeamStore={store.grafanaTeamStore}
+          onChange={this.handleFiltersChange}
+        />
+      </div>
+    );
+  }
+
+  handleFiltersChange = (filters: RemoteFiltersType, _isOnMount: boolean) => {
+    const { filtersStore } = this.props.store;
+    const currentTablePage = filtersStore.currentTablePageNum[PAGE.Users];
+
+    LocationHelper.update({ p: currentTablePage }, 'partial');
+
+    this.setState({ filters }, () => {
+      this.updateUsers();
+    });
+  };
+
+  applyFilters = () => {
```

Comment:
Seems this method is unused, can we remove it? Not related to your changes but there's another unused method in this component - `renderContacts`, we could remove it too